blocks/library/embed/index.js

```diff
@@ -85,8 +91,14 @@ registerBlockType( 'core/embed', {
 			this.noPreview = [
 				'facebook.com',
```

Comment:
I couldn't find much background on why we need to disable previews for some hosts. Should this be performed on the server instead of on the client? Also doesn't seem to need to be an instance variable assigned to each occurrence of the component but rather a top-level shared constant reference e.g. `PREVIEW_HOST_BLACKLIST`.